src/main/java/com/web01/animatch/service/AccountChangeService.java

```diff
@@ -0,0 +1,1530 @@
+ /**
+  * 小数点以下0の整数フォーマット
+  */
+ private static final String END_ZERO_DECIMAL_FORMAT = "^[+]?([1-9]{1})([0-9]{0,37})[.]?([0]{1,})$";
+ /**
+  * 営業時間フォーマット
+  */
+ private static final String BUSINESS_TIME_FORMAT = "^[0-9]{2}:[0-9]{2}$";
+ /**
+  * textarea開始タグ特殊文字
+  */
+ private static final String TEXTAREA_INIT_PART_TAG = "<textarea";
+ /**
+  * textarea終了タグ特殊文字
+  */
+ private static final String TEXTAREA_END_TAG = "</textarea>";
+
+ /**
+  * デフォルトコンストラクタ
+  */
+ public AccountChangeService() {
+  this.propertiesService = new PropertiesService();
+  this.messageService = new MessageService();
+ }
+
+ /**
+  * コンストラクタ
+  */
+ public AccountChangeService(String registedType) {
+  this.propertiesService = new PropertiesService();
+  registType = UserType.getEnumFromId(registedType);
+  this.messageService = new MessageService();
+
+  this.msgMap = new HashMap<>();
+ }
+
+ /**
+  * 登録区分ID取得
+  * @return 登録区分
+  */
+ private String getRegistTypeId() {
+  String registType = null;
+  switch (this.registType) {
+  //飼い主の場合
+  case OWNER:
+   registType = "001";
+   break;
+  //トリマーの場合
+  case TRIMMER:
+   registType = "002";
+   break;
+  default:
+   break;
+  }
+  return registType;
+ }
+
+ /**
+  * 登録成功失敗getter
+  * @return 登録成功失敗
+  */
+ public boolean isCanRegistFlg() {
+  return this.canRegist;
+ }
+
+ /**
+  * 初回表示設定
+  * @param request リクエストオブジェクト
+  */
+ public void setFirstDisplay(HttpServletRequest request) {
+  setInitPropertiesKey(request);
+  UserSession userSession = new UserSession();
+  SessionService sessionService = new SessionService((HttpServletRequest)request);
+  userSession = (UserSession)sessionService.getBindingKeySessionValue(AuthService.USER_SESSION_KEY_NAME);
+  Integer userId = userSession.getUserId();
+  Integer petId = userSession.getPetId();
+  Integer storeId = userSession.getStoreId();
+  setRegistedInfoAttributeKey(request, userId, petId, storeId);
+ }
+
+ /**
+  * 初期プロパティ設定
+  * @param request リクエストオブジェクト
+  */
+ private void setInitPropertiesKey(HttpServletRequest request) {
+  Map<String, String> registTypeMap = new HashMap<>();
+  Map<String, String> prefecturesMap = new HashMap<>();
+  Map<String, String> petTypeMap = new HashMap<>();
+  Map<String, String> weekdayMap = new HashMap<>();
+  Map<String, String> humanSexMap = new HashMap<>();
+  Map<String, String> petSexMap = new HashMap<>();
+  registTypeMap = this.propertiesService.getValues(PropertiesService.REGIST_TYPE_KEY_INIT_STR);
+  prefecturesMap = this.propertiesService.getValues(PropertiesService.PREFECTURES_KEY_INIT_STR);
+  petTypeMap = this.propertiesService.getValues(PropertiesService.PET_TYPE_KEY_INIT_STR);
+  weekdayMap = this.propertiesService.getValues(PropertiesService.WEEKDAY_KEY_INIT_STR);
+  humanSexMap = this.propertiesService.getValues(PropertiesService.HUMAN_SEX_KEY_INIT_STR);
+  petSexMap = this.propertiesService.getValues(PropertiesService.PET_SEX_KEY_INIT_STR);

```

Comment:
初期化後すぐに値を格納しているので、値の格納を初期値としてよいと思います。

```suggestion
// example
  Map<String, String> registTypeMap = this.propertiesService.getValues(PropertiesService.REGIST_TYPE_KEY_INIT_STR);
```

Reply:
上記、承知いたしました。修正いたします。